javascript/src/main/components/BasicCourseSearch/BasicCourseSearchForm.js

```diff
@@ -46,22 +49,33 @@ const BasicCourseSearchForm = ({ setCourseJSON, fetchJSON }) => {
 	};
 
 	const handleLevelOnChange = (event) => {
+        localStorage.setItem("BasicSearch.Level", event.target.value);
 		setLevel(event.target.value);
 	};
 
+    const handleQuarterOnChange = (quarter) => {
+        localStorage.setItem("BasicSearch.Quarter", quarter);
+        setQuarter(quarter);
+    }
+
+    const handleSubjectOnChange = (subject) => {
+        localStorage.setItem("BasicSearch.Subject", subject);
+        setSubject(subject);
+    }
+
 	return (
 		<Form onSubmit={handleSubmit}>
 			<SelectQuarter
 				quarters={quarters}
 				quarter={quarter}
-				setQuarter={setQuarter}
+				setQuarter={handleQuarterOnChange}
 				controlId={"BasicSearch.Quarter"}
 				label={"Quarter"}
```

Comment:
I wonder whether the local storage code could be moved inside the <SelectQuarter> component, using the value of either `label` or `controlId` as the lookup key?  

That bit of refactoring might allow us to put the code to do the local storage thing inside the `<SelectQuarter>` component, the `<SelectSubject>` component, and perhaps a new `<SelectCourseLevel>` component, so that this code doesn't have to be added to each page one at a time?

You don't need to do that in this PR.  But it could possibly be added as a future story.

There are are several stories here:
1. Factoring out the CourseLevel select into its own component.   That would be similar to the `SelectSubject` and `SelectQuarter` components, except easier since the options L, S, U, etc. could be hard coded.
2. Adding local storage logic into each of the three components <SelectSubject>, <SelectQuarter> and <SelectCourseLevel>.
3. Testing that this works on the various pages where those are used.
4. Removing the (now redundant) local storage logic from the pages that use those components.